### ISSUE TITLE
How to launch an application in maximized screen layout ?

### ISSUE THREAD

**Issue by Guytou11** [Botspot/autostar#4]
When launching an application with autostar, how to obtain a maximized screen automatically ?

**Comment by Botspot** [comment]
Autostar cannot help with this.
It depends on the application. For example, Chromium has a flag: `--start-maximized`, or something similar to that.

Other applications will have a fullscreen option, but many will not. This is when you will need to consult with the documentation of your Window Manager to see how to set up specific behaviors per application. For example, with Wayfire or LabWC, (the two options for default wayland compositor in PiOS Bookworm) they have documentation for how to add specific actions (like maximizing) to specific app-ids.

Be aware that I have never needed to do this, and I cannot help you much further. I encourage you to look at the documentation for Wayfire or LabWC to see how to do it, and with more specific questions check their issues on github or start a new one. Also, there is always the Raspberry Pi Forums, where questions like yours are common.

TL;DR Some applications make it easy to start maximized or fullscreen, while the rest will need to be maximized by the window manager.
